composer.json

```diff
@@ -18,7 +18,7 @@
 		"contributte/application": "^0.5.0",
 		"contributte/di": "^0.5.0",
 		"contributte/cache": "^0.6.0",
-		"contributte/http": "^0.4.0",
+		"contributte/http": "0.5.x-dev",
```

Comment:
Waiting for https://github.com/contributte/http/commit/f1872fe39e04ea786ce79af9da333735a6c8ce44 to get into stable version.

Reply:
https://github.com/contributte/http/releases/tag/v0.4.1